Product/Production/Common/CONNECTCoreLib/src/test/java/gov/hhs/fha/nhinc/messaging/service/decorator/HttpHeaderServiceEndpointDecoratorTest.java

```diff
@@ -101,6 +101,31 @@ protected PropertyAccessor getPropertyAccessor() {
     }
     
     @Test
+    public void testConfigure_WithoutValues() throws PropertyAccessException {
+        CONNECTTestClient<TestServicePortType> testClient = new CONNECTTestClient<>(
+                new TestServicePortDescriptor());
+
+        ServiceEndpoint<TestServicePortType> serviceEndpoint = testClient.getServiceEndpoint();
+        
+        final PropertyAccessor mockPropAccessor = mock(PropertyAccessor.class);
+
+        AssertionType assertion = new AssertionType();
+        assertion.setKeepAlive("true");
+        
+        CONNECTCustomHttpHeadersType assertionHeaders = new CONNECTCustomHttpHeadersType();
+        assertionHeaders.setHeaderName("headerName");
+        assertionHeaders.setHeaderValue("headerValue");
+        
+        assertion.getCONNECTCustomHttpHeaders().add(assertionHeaders);
+
+        HttpHeaderServiceEndpointDecorator headerDecorator = new HttpHeaderServiceEndpointDecorator(serviceEndpoint, assertion); 
+
+    	headerDecorator.configure();
+        
+        validateConfiguration(headerDecorator); 
```

Comment:
Does this test pass?  Shouldn't throwing the property exception cause it not to add the headers to the binding provider?